icons/shield_us_us_historic_2.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These icons might have a generic use in some Latin American countries. Consider a more generic filename like `badge_crossbar`.

Reply:
Done. shield_badge_crossbar_{2,3}.svg is now black on white like shield_badge_{2,3}.svg, and this network’s definition recolors it to brown.

When we get around to implementing Bolivia’s federal routes and Paraguay’s primary routes, we’ll be able to recolor these images to white on green. Some other countries have similar-looking shields that aren’t similar enough to reuse these images:

* Chile’s national routes are shaped differently at the top: https://github.com/ZeLonewolf/openstreetmap-americana/pull/474#discussion_r913828239.
* The Pan-American Highway shield is inspired by the old U.S. Route shield, but it’s defaced with a map of the Americas: https://github.com/ZeLonewolf/openstreetmap-americana/issues/421#issuecomment-1159569738.
* Peru’s shields are stretched vertically.